src/Tabs.jsx

```diff
@@ -12,6 +12,10 @@ class Tabs extends React.Component {
 				{'tabs-tab--selected': tab.props.isSelected}
 			);
 
+			// remove prop that is not standard DOM
+			// before rendering
+			delete tab.props.isSelected;
```

Comment:
I'm trying to think if there's another way to do this but this seems like the simplest way. LGTM after tests pass.